src/SapNwRfc/Internal/OutputMapper.cs

```diff
@@ -120,10 +125,21 @@ public static TOutput Extract<TOutput>(RfcInterop interop, IntPtr dataHandle)
             // ReSharper disable once PossibleNullReferenceException
             PropertyInfo fieldValueProperty = extractMethod.ReturnType.GetProperty(nameof(Field<object>.Value));
 
+            Collection<Expression> arguments = new Collection<Expression>() { interop, dataHandle, name };
```

Comment:
```suggestion
            var arguments = new Collection<Expression> { interop, dataHandle, name };
```

Convention: when the type is clear from the right-side of the assignment operator, there's no need to repeat the type on the left-side. Also, I think you can drop the `()`.